samples/sqs/simple/sample.md

```diff
@@ -21,9 +21,12 @@ Add the [AWS Access Key ID and AWS Secret Access Key](http://docs.aws.amazon.com
 
  * Access Key ID in `AWS_ACCESS_KEY_ID`
  * Secret Access Key in `AWS_SECRET_ACCESS_KEY`
+ * Default Region in `AWS_DEFAULT_REGION`
 
 See also [AWS Account Identifiers](http://docs.aws.amazon.com/general/latest/gr/acct-identifiers.html), [Managing Access Keys for a AWS Account](http://docs.aws.amazon.com/general/latest/gr/managing-aws-access-keys.html), and [IAM Security Credentials](https://console.aws.amazon.com/iam/home#/security_credential).
 
+See also [AWS Regions](http://docs.aws.amazon.com/general/latest/gr/rande.html) for available regions to configure.
```

Comment:
`for a list of available regions to configure.` or even simpler `for a list of available regions.`